src/org/usfirst/frc/team3182/robot/DriveTrain.java

```diff
@@ -15,31 +16,39 @@
 	private RobotDrive drive;
 	private Encoder leftEncoder, rightEncoder;
 	private Talon leftController, rightController;
+	private PIDController leftPIDController, rightPIDController;
 	
 	/**
 	 * DriveTrain constructor, creates a DriveTrain
 	 */
 	public DriveTrain(){
 		leftController = new Talon(RobotConfig.driveWheelL);
 		rightController = new Talon(RobotConfig.driveWheelR);
+		//leftController.setInverted(true);
+		//rightController.setInverted(true);
```

Comment:
The correct value for setInverted() should be reviewed later when we understand why deploys to the roboRio did not appear to be taking